Stop removing blank lines in .ts files

smokris opened the pull request:
Fixes issue where po2ts would remove consecutive linebreaks in source and translated strings.  Added tests.

(I'm not sure why the blank-line removal code was there in the first place; I traced it back as far as 9836716 (12 years ago) and it isn't documented.  Removing it doesn't break any tests.)

nijel commented on translate/storage/test_ts2.py:
```diff
@@ -60,6 +60,12 @@
+
+Line 3</source>
+        <translation type="unfinished"></translation>
+    </message>
```
You're adding test for ts2 storage, but patching ts storage, that looks like a mismatch.

smokris commented:
> use ts2 instead of ts in the converters

Ah, good point.  In the latest version of this PR, I reworked `po2ts` to use `ts2` (and removed the changes to `ts`).

nijel commented:
Merged, thanks for your contribution!
